projects/kit/components/accordion/accordion-item/accordion-item-eager-content.directive.ts

```diff
@@ -0,0 +1,16 @@
+import {ChangeDetectorRef, Directive, Inject, Self, TemplateRef} from '@angular/core';
+import {PolymorpheusTemplate} from '@tinkoff/ng-polymorpheus';
+
+@Directive({
+    selector: '[tuiAccordionItemEagerContent]',
```

Comment:
I like this idea, how about we keep the same selector but write here `[tuiAccordionItemContent]:not(ng-template)` and on the other, lazy directive we will change selector to `ng-template[tuiAccordionItemContent]`. This way people will not need to remember two selectors, it will be the same but for eager you put it on, say, a `div` and for lazy content you put it on a `ng-template` (it would also work with a star syntax: `<div *tuiAccordionItemContent>` for lazy and `<div tuiAccordionItemContent>` for eager).

Reply:
Updated directive selectors to suggested